com.unity.render-pipelines.high-definition/Documentation~/Compositor-User-Guide.md

```diff
@@ -1,8 +1,12 @@
 # Using the Graphics Compositor
 To open the Graphics Compositor window in the Unity Editor, select **Window > Render Pipeline > Graphics Compositor** from the menu. From this window, you can enable the Graphics Compositor and control its configuration. The first time you enable the Graphics Compositor, it automatically uses a default *"pass-through"* composition profile that forwards the output of the main [Camera](HDRP-Camera.md) to the final composed frame. You can add more [Layers](#layer-types) to this profile and edit their properties or you can load another profile from disk. For information on the properties in the Compositor window, see [Compositor window](Compositor-User-Options.md).
 
+## General Workflow
+- The composition output is shown in the **Game View**, while the scene can be edited in **Scene View**. To have a live preview of the output, it is recommended to have the **Scene View** and **Game View** side-by-side, or undock the gameview from the Editor and put it on a separate monitor (if available).
```

Comment:
- The composition output render in the Game view, while you can edit the scene in the Scene view. To have a live preview of the output, it is best practice to have the Scene view and Game view side-by-side, or undock the Game view from the Editor and put it on a separate monitor (if available).